floris/core/flow_field.py

```diff
@@ -281,29 +283,45 @@ def generate_heterogeneous_wind_map(self):
                 - **y**: A list of y locations at which the speed up factors are defined.
                 - **z** (optional): A list of z locations at which the speed up factors are defined.
         """
-        speed_multipliers = self.heterogeneous_inflow_config['speed_multipliers']
+        speed_multipliers = np.array(self.heterogeneous_inflow_config['speed_multipliers'])
         x = self.heterogeneous_inflow_config['x']
         y = self.heterogeneous_inflow_config['y']
         z = self.heterogeneous_inflow_config['z']
 
+        interps_f = [] # Declare an empty list to store interpolants by findex
```

Comment:
Just a heads up that we generally try to use Numpy arrays over Python lists, unless there's a specific reason. Using a consistent array data structure everywhere means we don't have to guess whether something is a List or np.array.

Reply:
Good point @rafmudaf . `interps_f`, once it is filled, gets assigned to `self.het_map`, which is [declared as a list here](https://github.com/NREL/floris/blob/fd0e599ef01dc9f1d426d2b4c99e2f6beb57fefd/floris/core/flow_field.py#L44). 

However, I have tested instead declaring `interps_f` as a Numpy array (`interps_f = np.empty(self.n_findex, dtype=object)`) and then assigning the individual interps via index rather than `append`. This changes the type of  `self.het_map` to an array, but all tests pass and relevant examples run. Do you suggest we change the attrs declaration to be type `NDArray(Float?)` (the Float part doesn't seem right)?

Reply:
Ah, never mind---I see that we have an `NDArrayObject` type, I'll change the `het_map` declaration to be of that type.

Reply:
Ah hm, it seems like the type hint will be incorrect at one time or another if it goes from List on init to Numpy array. Sorry, this seems like a more complicated suggestion than I intended.

Reply:
Actually, never mind, I think I misunderstood. Your last commit looks good to me!